app/reducers/views/channel.js

```diff
@@ -322,6 +322,13 @@ function lastChannelViewTime(state = {}, action) {
 function keepChannelIdAsUnread(state = null, action) {
     switch (action.type) {
     case ChannelTypes.SELECT_CHANNEL: {
+        if (!action.extra && action.data) {
```

Comment:
Just curious about this - `!action.extra` checks for truthiness. Which means if the property is set to an empty string or it is `false`, then the check will be true. https://developer.mozilla.org/en-US/docs/Glossary/Falsy.

Should we explicitly check if `extra` is not a property of `action` ?

Reply:
Lets say is an empty string or false or anything other than an object, it will still be defined and then the second if below will handle it.

Reply:
Ok, I guess that's one way to take care of this.